src/RequestHandlerSwooleRunner.php

```diff
@@ -193,24 +206,101 @@ function (Throwable $exception) use ($serverRequestErrorResponseGenerator) : Res
             ));
         }
 
-        $this->logger = $logger ?: new StdoutLogger();
+        $this->logger = $logger ? : new StdoutLogger();
+        $this->pidManager = $pidManager;
     }
 
     /**
      * Run the application
      */
     public function run() : void
     {
-        $this->swooleHttpServer->on('start', [$this, 'onStart']);
-        $this->swooleHttpServer->on('request', [$this, 'onRequest']);
-        $this->swooleHttpServer->start();
+        $opts = new Getopt([
+            'daemonize|d'  => 'daemonize the swoole server process',
+            'worker_num|n=i' => 'set worker nums',
+            'dispatch_mode=i' => 'set dispatch_mode',
```

Comment:
Detail what valid modes might be in the help message.

Reply:
I've removed `dispatch_mode` option for start command, I think nobody will change dispatch mode dynamically, dispatch mode will be confirmed in development stage